generate_tools.go

```diff
@@ -14,5 +14,6 @@ import (
 	_ "github.com/mailru/easyjson/easyjson"
 	_ "github.com/shuLhan/go-bindata/cmd/go-bindata"
 	_ "github.com/tinylib/msgp"
+	_ "github.com/vektra/mockery/v2"
```

Comment:
Is this really a tool used in `go:generate`? If not it should go on `internal/tools/tools.go` and be installed via `inv install-tools`

Reply:
The tool is run using `go run github.com/vektra/mockery/v2` in a script (pkg/compliance/genso the idea was that it uses the version specified in go.mod
I don't know if the internal/tools/go.mod will be used in this case

Reply:
I modified internal/tools/go.mod and use the `mockery` binary instead of `go run`

Reply:
Thanks! That way it doesn't end up in the licenses list :)